common/src/main/java/org/wso2/carbon/testgrid/common/InfrastructureProvider.java

```diff
@@ -47,7 +47,7 @@
      *                       that should be created.
      * @param infraRepoDir - Location of the cloned repository related to infrastructure.
      * @return Deployment -  Deployment object including the created host, ip details
-     * @throws TestGridInfrastructureException
+     * @throws TestGridInfrastructureException - If something goes wrong during the infrastructure creation process.
```

Comment:
Better to say "thrown when error occurs in the infrastructure creation process"

Reply:
Fixed with 14598a57750f800c12e6cb5a3828474056e7535c.